docs/literate/src/files/shock_capturing.jl

```diff
@@ -225,6 +225,137 @@ sol = solve(ode, CarpenterKennedy2N54(stage_limiter!, williamson_condition=false
 using Plots
 plot(sol)
 
+# # Entropy bounded limiter
+
+# As argued in the description of the positivity preserving limiter above it might sometimes be
+# necessary to apply advanced techniques to ensure a physically meaningful solution.
+# Apart from the positivity of pressure and density, the entropy of the system should increase 
```

Comment:
Something like
```suggestion
# Apart from the positivity of pressure and density, the (unphysical or mathematical) entropy of the system should increase 
```
?

Reply:
So this is actually the physical entropy. I will add this to clarify, thanks!

Reply:
Oh okay :+1: But in `src/callbacks_stage/entropy_bounded_limiter.jl` this

> This is an ideal-gas specific limiter that bounds the (unphysical) entropy decrease per element  from one time step (or Runge-Kutta stage) to the next.

 is correct?

Reply:
Yeah, here "unphysical" addresses "decrease", not "entropy" :grimacing:

Reply:
I will add maybe again that it is the thermodynamic /physical entropy :sweat_smile: